internal/internal_workflow.go

```diff
@@ -215,6 +215,7 @@ type (
 		WaitForCancellation      bool
 		WorkflowIDReusePolicy    enumspb.WorkflowIdReusePolicy
 		WorkflowIDConflictPolicy enumspb.WorkflowIdConflictPolicy
+		OnConflictOptions        *OnConflictOptions
```

Comment:
nit: can we add a comment explaining these are only used for the test environment?